corrector.rb

```diff
@@ -0,0 +1,5 @@
+class Corrector
+  def correct_name(name)
+    name.slice(0, 10).capitalize
```

Comment:
Great work on creating the `Corrector` class with `correct_name` method... 💯 
only one thing i want to mention is 

 - `capitalize` returns a new string with the first letter capitalized
 - `capitalize!` modifies or mutates the original string 

For further read [LINK](https://stackoverflow.com/questions/26103869/using-capitalize-or-capitalize)


- [ ] Change `capitalize` to `capitalize!` to mutate the user input name to capitalize it if it's not. 👍